### DISCUSSION TITLE
Monika not being able to read my gifts

### DISCUSSION THREAD

**Discussion by Tok0yo** [Monika-After-Story/MonikaModDev#9827] [QA - tech support + general]
I've been dealing with this problem ever since i changed pcs, I reinstalled her many times yet she can't read them still.

**Comment by Wingdinggaster656** [comment]
What is the gift that you are trying to give? Give the gift file name.

Also, you said "can't read them", it's that she noticed them, simply say that she can not understand them, or she didn't do any reaction to them at all?

**Comment by Tok0yo** [comment]
I've been trying to gift her clothes, but she can read the gift files implanted in the game. But when it comes to custom stuff, she has trouble reading it. And yes she couldn't understand them, she would always say "I'm sorry (player) But for some reason, I can't read the file." I don't know if it's because of the way I installed it or a big error in my game.

**Comment by Wingdinggaster656** [comment]
As what I said:

> Give the gift name.

"Gift her clothes" isn't that exact. Give us the gift file name.

**Comment by Tok0yo** [comment]
Hello! Sorry I haven't been replying that quickly but as I mean't by giving
her a gift, I mean't by when I put in the gift name let's say
formaldress.gift in the characters folder she can't read it. Most gift
files I putted in is smth she can't read. ^^

**Comment by Booplicate** [comment]
For installing spritepacks follow [this video](https://www.youtube.com/watch?v=bjJ0U8l1m8c). And [this guide](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#troubleshooting-installation).
